docs/activities/meetings/index.html

```diff
@@ -5,69 +5,25 @@
 ---
 <div>
     <h2>W3C WoT – Web Conference Meetings</h2>
-    
+	
 <script>
-function initTimezone() {
-    var tz = "UTC"; // default
-    // try to autodetect
-    var myDate = new Date();
-    var diffInMinutes = myDate.getTimezoneOffset();
-    console.log(diffInMinutes)
-    if (diffInMinutes) {
-        switch(diffInMinutes) {
-            case 480:
-                tz = "UTC-8";
-            case 420:
-                tz = "UTC-7";
-            case 300:
-                tz = "UTC-5";
-            case 240:
-                tz = "UTC-4";
-            case 180:
-                tz = "UTC-3";
-                break;
-            case 0:
-                tz = "UTC+0";
-                break;
-            case -60:
-                tz = "UTC+1";
-                break;
-            case -120:
-                tz = "UTC+2";
-                break;
-            case -180:
-                tz = "UTC+3";
-                break;
-            case -480:
-                tz = "UTC+8";
-                break;
-            case -540:
-                tz = "UTC+9";
-                break;
-        }
-    }
-    document.getElementById("timezoneSelect").value = tz;
-}
 function updateTimezone() {
   var selVal = document.getElementById("timezoneSelect").value;
   
-  // hide "all"
-  document.getElementById("UTC-7").style.display = 'none';
-  // document.getElementById("UTC-5").style.display = 'none';
-  document.getElementById("UTC-4").style.display = 'none';
-  // document.getElementById("UTC").style.display = 'none';
-  document.getElementById("UTC+1").style.display = 'none';
-  document.getElementById("UTC+2").style.display = 'none';
-  document.getElementById("UTC+3").style.display = 'none';
-  document.getElementById("UTC+8").style.display = 'none';
-  document.getElementById("UTC+9").style.display = 'none';
+  // Options:
+  // * tab=week or tab=day etc
+  // * start_of_week=work // Mo-Fr
+  // * target=_blank // Links
+  var newSrc = "https://open-web-calendar.hosted.quelltext.eu/calendar.html?url=https://www.w3.org/groups/wg/wot/calendar/export&start_of_week=work";
   
-  // set the selected one visible
-  if (document.getElementById(selVal)) {
-    document.getElementById(selVal).style.display = '';
-  } else {
-    document.getElementById("UTC").style.display = ''; // default
+  // XXX should be removed
+  newSrc += "&tab=week&starting_hour=11"; // to make it work properly we need to adjust the hours based on the timezone
```

Comment:
While the upstream issue is being discussed, can't we use the previous timezone finding logic to enter the starting hours dynamically?

Reply:
That is also true and makes everything much simpler :D

Reply:
It does look pretty indeed.